appveyor.yml

```diff
@@ -41,6 +41,7 @@ build_script:
 # Will figure out and fix it in separate PR.
 #- cmd: C:\Miniconda36\Scripts\backend-test-tools generate-data
 - cmd: git diff --exit-code
+- cmd: ! grep -R --include='*.cc' --include='*.h' 'namespace onnx' .
```

Comment:
@linkerzhang Do you know how can I express this in windows? :-) I want to forbid directly hardcoding "namespace onnx" in all c++ source code.

Reply:
Try findstr command?